Index: src/themes/openems/environments/edge-prod.ts
===================================================================
--- src.orig/themes/openems/environments/edge-prod.ts
+++ src/themes/openems/environments/edge-prod.ts
@@ -5,7 +5,7 @@ export const environment: Environment =
     ...theme, ...{
 
         backend: 'OpenEMS Edge',
-        url: "ws://" + location.hostname + ":8075",
+        url: "ws://localhost:8075",
 
         production: true,
         debugMode: false,
