Index: environments/backend-prod.ts
===================================================================
--- environments.orig/backend-prod.ts
+++ environments/backend-prod.ts
@@ -5,7 +5,7 @@ export const environment: Environment =
     ...theme, ...{
 
         backend: 'OpenEMS Backend',
-        url: "ws://" + location.hostname + ":8082",
+        url: "ws://localhost:8082",
 
         production: true,
         debugMode: false,
