Index: /src/themes/openems/environments/backend-prod.ts
===================================================================
--- /src.orig/themes/openems/environments/backend-prod.ts
+++ /src/themes/openems/environments/backend-prod.ts
@@ -5,7 +5,7 @@ export const environment: Environment =
     ...theme, ...{
 
         backend: 'OpenEMS Backend',
-        url: "ws://" + location.hostname + ":8082",
+        url: window["env"]["websocket"],
 
         production: true,
         debugMode: false,
